### PR TITLE
feat(list-item): Add content-bottom slot for placing content below the label and description of the component

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #8173

## Summary

- This is a good starting point. We just need input from design. cc @macandcheese 
- Adds a new slot `content-bottom`. Could also be called `content-block-end`?
- `content-bottom` is indented the same as the item.
- screenshot tests

### CODE REVIEW

**Comment by macandcheese** [comment]
Nice! No issue with the visuals but cc @SkyeSeitz @ashetland for visibility. I think we need to do a larger pass on the "inline/bottom/block/top" stuff, but for now maybe "bottom" is clearer since we already have "content/actions-end" and those don't use "inline".

**Comment by ashetland** [comment]
> Nice! No issue with the visuals but cc @SkyeSeitz @ashetland for visibility. I think we need to do a larger pass on the "inline/bottom/block/top" stuff, but for now maybe "bottom" is clearer since we already have "content/actions-end" and those don't use "inline".

I vote for `content-bottom`. Follows Modal which has `content`, `-top`, `-bottom`.

> I think we need to do a larger pass on the "inline/bottom/block/top" stuff

Agree.

**Comment by ashetland** [comment]
I think I know the answer to this, but it ties into other things I'm working on. Should the bottom slot have some default padding on it or do we leave it up to the user? At first I was thinking about x-padding to help align it with the heading+description, but then of course that leads to asking about y-padding too.

**Comment by SkyeSeitz** [comment]
Agreed on `content-bottom`. Just did a quick pass and I believe Card's thumbnailPosition is now the only prop still using block-end/start language, which we could consider changing to follow the top/bottom paradigm moving forward.

**Comment by macandcheese** [comment]
I do think inline/block-start/end (as in Card) is a more accurate verbiage that aligns with css terminology and is writing mode agnostic. 

But in our current nomenclature here it wouldn’t fit, so bottom IMO is good for now.

Places like Shell, other components that use top/bottom/start/end, maybe can all use inline/block-start/end in the future.

**Comment by driskull** [comment]
@jcfranco @macandcheese can I get a review of this one?

**Review by macandcheese — APPROVED** [review]
Awesome! Can you add this slot to the jsdoc definitions at the top of `list-item.tsx` so it comes through in docs? Looks good otherwise, we can work to add "good use cases" to the doc site.

**Comment by driskull** [comment]
Nice catch. Added slot doc. @geospatialem can you review it?

**Comment by jcfranco** [comment]
> I do think inline/block-start/end (as in Card) is a more accurate verbiage that aligns with css terminology and is writing mode agnostic.

I disagree here. The `inline/block`-related CSS logical prop terminology is very much `writing-mode` dependent. All logical props have something similar to this in their description:

> It corresponds to the top, right, bottom, or left property depending on the values defined for writing-mode, direction, and text-orientation.

 ☝️from https://developer.mozilla.org/en-US/docs/Web/CSS/inset-inline-end

**Comment by jcfranco** [comment]
Scratch what I said above. I read that wrong. 😅

nothing-to-see

**Comment by jcfranco** [comment]
> I think we need to do a larger pass on the "inline/bottom/block/top" stuff, but for now maybe "bottom" is clearer since we already have "content/actions-end" and those don't use "inline".

> I do think inline/block-start/end (as in Card) is a more accurate verbiage that aligns with css terminology and is writing mode agnostic.

@macandcheese Do we need an audit for this for the future? Seems like we should for consistency.

File: packages/calcite-components/src/components/list-item/list-item.tsx [hunk 1]
```diff
@@ -51,6 +51,7 @@ import {
  * @slot content - A slot for adding non-actionable, centered content in place of the `label` and `description` of the component.
  * @slot content-end - A slot for adding non-actionable elements after the label and description of the component.
  * @slot actions-end - A slot for adding actionable `calcite-action` elements after the content of the component.
+ * @slot content-bottom - A slot for adding content below the label and description of the component.
```
Comment (geospatialem) [inline packages/calcite-components/src/components/list-item/list-item.tsx:54]: Awesome! Small recommended tweak and consistency with backticks in to differentiate the props:

```
A slot for adding content below the component's `label` and `description`.
```

**Pull request merged**